doc/troubleshooting.txt

```diff
@@ -0,0 +1,368 @@
+.. _troubleshooting:
+
+Troubleshooting
+###############
+
+Here are Linux troubleshooting instructions. There is a specific `MacOS`_ section.
+
+- :ref:`network_error_proxy`
+- :ref:`PYTHONPATH`
+- :ref:`slow_or_memory`
+- :ref:`TensorVariable_TypeError`
+- :ref:`out_of_memory`
+- :ref:`float64_output`
+- :ref:`test_theano`
+- :ref:`test_BLAS`
+
+.. _network_error_proxy:
+
+Why do I get a network error when I install Theano
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are behind a proxy, you must do some extra configuration steps
+before starting the installation. You must set the environment
+variable ``http_proxy`` to the proxy address. Using bash this is
+accomplished with the command
+``export http_proxy="http://user:pass@my.site:port/"``
+You can also provide the ``--proxy=[user:pass@]url:port`` parameter
+to pip. The ``[user:pass@]`` portion is optional.
+
+.. _PYTHONPATH:
+
+How to configure the PYTHONPATH
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This should not be necessary if Theano was installed through `pip install`, like the updated instructions state.
I would be in favour of removing that section.